src/common/history.h

```diff
@@ -76,6 +84,15 @@ char *dt_history_get_items_as_string(int32_t imgid);
 /* check if a module exists in the history of corresponding image */
 gboolean dt_history_check_module_exists(int32_t imgid, const char *operation);
 
+/** calculate the hash for that history from db*/
+gsize dt_hash_history_compute_from_db(const int32_t imgid, guint8 **hash);
```

Comment:
convention is that all routine in one module just start with dt_<module>_ so here it should all be:
`dt_history_hash...();`

For the three new routines.

Reply:
There is a dt_history_xxx, dt_dev_history_xxx, it's why I've added a dy_hash_history_xxx.

For the routine called only in one module I've seen that `dt_` prefix is replaced by `_`. So if that the subject of the remark, I should rename `dt_hash_history_compute_from_db()` to `_hash_history_compute_from_db()`. is that correct ? If not, I haven't understood the remark.

Reply:
done, both.

Reply:
@phweyland ; Thanks, not that dt_dev_history exists indeed, but in develop.h which makes my point :)